portal-web/test/functional/com/liferay/portalweb/macros/LiferayEvent.macro

```diff
@@ -75,4 +75,31 @@ window.LFR_JS_EVENTS_LOG = {
 		RunScript(locator1 = "${javascript}");
 	}
 
+	macro waitForLiferayEvent {
+		var javascript = '''
+try {
```

Comment:
Not sure where to comment, but `Add macro similar to MetalComponent.waitForMetalComponent` doesn't feel like a descriptive enough commit message. What's so relevant about `waitForMetalComponent` that wins over a simple explanation of what's being done here? Is it that everyone knows that macro and what it does by heart?

It'd be better to simply state what's being done here and why. That way, people reviewing or retracing our steps would understand why we did what we did.